### ISSUE TITLE
Addon doesn't start

### ISSUE THREAD

**Issue by neonline78** [Minims/SomfyProtect2MQTT#39]
Hello everyone,
The addon doesn't start without apparently reason, also after resume old backup.
My installation is home assistant os under vm.
The log say:

s6-overlay-suexec: fatal: can only run as pid 1

Any help?
Thank you

**Comment by About2Break** [comment]
It's due to the following change: https://developers.home-assistant.io/blog/2022/05/12/s6-overlay-base-images/

I tried adding init: false to the config, still the same error.

**Comment by Minims** [comment]
Hello, thanks for the link about the change.
I can't do it for now, but it seems is needs a rework on the current add-on to avoid this issue. 
I may have a look next week, unless if @schumijo can have a look before

**Comment by yobrx** [comment]
I have the same problem too 😔

Home Assistant Core 2022.5.4
Home Assistant Supervisor 2022.05.2
Kernel version
Agent version

**Comment by neonline78** [comment]
> Hello, thanks for the link about the change. I can't do it for now, but it seems is needs a rework on the current add-on to avoid this issue. I may have a look next week, unless if @schumijo can have a look before

Thank you for your job! I have temporary solved with a installation of homebridge with somfy protect plug in, waiting for solution...

**Comment by schumijo** [comment]
Fix will be available tomorrow ;)

**Comment by PoVPoV29** [comment]
Hello,
Issue seems to be happening again 

`2022-05-17 21:14:51,197 [INFO] [__main__:48] Starting SomfyProtect2Mqtt
2022-05-17 21:14:51,198 [INFO] [root:40] Reading config file /run/s6/legacy-services/example/somfyProtect2Mqtt/config/config.yaml
2022-05-17 21:14:51,199 [ERROR] [utils:42] File /run/s6/legacy-services/example/somfyProtect2Mqtt/config/config.yaml not found
2022-05-17 21:14:51,199 [INFO] [root:110] Init SSO
Traceback (most recent call last):
  File "/usr/bin/SomfyProtect2MQTT/somfyProtect2Mqtt/main.py", line 52, in <module>
    SSO = init_sso(config=CONFIG)
  File "/usr/bin/SomfyProtect2MQTT/somfyProtect2Mqtt/somfy_protect/sso/__init__.py", line 111, in init_sso
    username = config.get("somfy_protect").get("username")
AttributeError: 'NoneType' object has no attribute 'get'
[19:14:51] WARNING: Halt add-on
s6-rc: info: service legacy-services: stopping
[19:14:51] INFO: Service restart after closing
[19:14:51] INFO: Service restart after closing
s6-rc: info: service legacy-services successfully stopped
s6-rc: info: service legacy-cont-init: stopping
s6-rc: info: service legacy-cont-init successfully stopped
s6-rc: info: service fix-attrs: stopping
s6-rc: info: service fix-attrs successfully stopped
s6-rc: info: service s6rc-oneshot-runner: stopping
s6-rc: info: service s6rc-oneshot-runner successfully stopped`

**Comment by Minims** [comment]
> Issue seems to be happening again

I know, I'm currently working on it and try to make it work.

**Comment by neonline78** [comment]
> Hello, Issue seems to be happening again
> 
> `2022-05-17 21:14:51,197 [INFO] [__main__:48] Starting SomfyProtect2Mqtt 2022-05-17 21:14:51,198 [INFO] [root:40] Reading config file /run/s6/legacy-services/example/somfyProtect2Mqtt/config/config.yaml 2022-05-17 21:14:51,199 [ERROR] [utils:42] File /run/s6/legacy-services/example/somfyProtect2Mqtt/config/config.yaml not found 2022-05-17 21:14:51,199 [INFO] [root:110] Init SSO Traceback (most recent call last): File "/usr/bin/SomfyProtect2MQTT/somfyProtect2Mqtt/main.py", line 52, in <module> SSO = init_sso(config=CONFIG) File "/usr/bin/SomfyProtect2MQTT/somfyProtect2Mqtt/somfy_protect/sso/__init__.py", line 111, in init_sso username = config.get("somfy_protect").get("username") AttributeError: 'NoneType' object has no attribute 'get' [19:14:51] WARNING: Halt add-on s6-rc: info: service legacy-services: stopping [19:14:51] INFO: Service restart after closing [19:14:51] INFO: Service restart after closing s6-rc: info: service legacy-services successfully stopped s6-rc: info: service legacy-cont-init: stopping s6-rc: info: service legacy-cont-init successfully stopped s6-rc: info: service fix-attrs: stopping s6-rc: info: service fix-attrs successfully stopped s6-rc: info: service s6rc-oneshot-runner: stopping s6-rc: info: service s6rc-oneshot-runner successfully stopped`

Tomorrow it's not today ;)

**Comment by Minims** [comment]
This version should work : 
Addon version :  0.1.9.8
repo https://github.com/Minims/homeassistant-addons

[![Open your Home Assistant instance and show the add add-on repository dialog with a specific repository URL pre-filled.](https://my.home-assistant.io/badges/supervisor_add_addon_repository.svg)](https://my.home-assistant.io/redirect/supervisor_add_addon_repository/?repository_url=https%3A%2F%2Fgithub.com%2FMinims%2Fhomeassistant-addons)

**Comment by About2Break** [comment]
Works great! Thanks for you very fast fix!

**Comment by neonline78** [comment]
Works perfect! thank you!

**Comment by schumijo** [comment]
> This version should work : Addon version : 0.1.9.8 repo https://github.com/Minims/homeassistant-addons
> 
> [![Open your Home Assistant instance and show the add add-on repository dialog with a specific repository URL pre-filled.](https://camo.githubusercontent.com/9e0c22cf56020b6ad057d06b87690b7a3ae14d8ada8b16a51abf6e8ab97df755/68747470733a2f2f6d792e686f6d652d617373697374616e742e696f2f6261646765732f73757065727669736f725f6164645f6164646f6e5f7265706f7369746f72792e737667)](https://my.home-assistant.io/redirect/supervisor_add_addon_repository/?repository_url=https%3A%2F%2Fgithub.com%2FMinims%2Fhomeassistant-addons)

Hi @Minims 

I was doing the same modification on my repo but not enough time to complete yesterday (I'm a young father for few days so don't have a lot of free time :) )
I think everybody should move to your repo and close mine as you are the dev of this excellent project.

**Comment by enzopanzani** [comment]
This patch allows SomfyProtect2mqtt to work again. 
A big thank you to the developers for their work and their reactivity.
_@schumijo: Congratulations on your child!_

**Comment by Minims** [comment]
> I was doing the same modification on my repo but not enough time to complete yesterday (I'm a young father for few days so don't have a lot of free time :) )
> I think everybody should now move to your repo and close mine as you are the dev of this excellent project. If you need help about addons, I'm available ;)

Hi @schumijo,
Congratulations ! 

I have used the example-addon to make it work again. It was a bit difficult to find the time, but as it was not working anymore, I have tried to restore the service for all users quickly.

It's Ok for me if you close your repo. 
Feel free to make a PR on mine for any improvement. 

Can we find a solution to help the current users of your addon to migrate on mine ? 
A new version, like an hello world, with the message to migrate to a new repo ? Any better idea ?
